contracts/scripts/native_solc_compile_all_vrfv2plus

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
is this needed? this is already in https://github.com/smartcontractkit/chainlink/blob/develop/contracts/scripts/native_solc_compile_all_vrf

Reply:
the naming convention here is defined [here](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/GNUmakefile#L65) . I don't think maintaining a separate foundry profile (e.g. vrfv2plus) is needed